Mathlib/Algebra/FreeMonoid/Symbols.lean

```diff
@@ -0,0 +1,35 @@
+/-
+Copyright (c) 2024 Hannah Fechtner. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Hannah Fechtner
+-/
+
+import Mathlib.Algebra.FreeMonoid.Basic
+import Mathlib.Data.Finset.Basic
+
+/-!
+# The finite set of symbols in a FreeMonoid element
+
+This is separated from the main FreeMonoid file, as it imports the finiteness hierarchy
+-/
+
+variable {α : Type*} [DecidableEq α]
+
+open FreeMonoid
+
+/-- the set of unique symbols in a free monoid element -/
+def symbols (a : FreeMonoid α) : Finset α := List.toFinset a
+
+@[to_additive (attr := simp)]
```

Comment:
I'm confused what `to_additive` does here, if `symbols` itself hasn't been `to_additive`ized.
Could you please add the attribute to all declarations in this file?

Reply:
If some definition hasn't been additivised, then it is simply kept invariant when translating

Reply:
I messed up - I used open instead of namespace, and then to resolve the errors, I fussed with the `to_additive`

It's fixed now! (thanks to adomani)